create_archive.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# strict mode
+set -eu
+
+# getting version of the package
+version=$(python setup.py --version)
+echo "Creating archive for dakara_feeder v$version"
+
+# install twine
+pip install --upgrade twine
+
+# clean the dist directory
+rm -rf dist/*
+
+# create the distribution packages
+python setup.py sdist bdist_wheel
+
+# upload to PyPI
+echo "Package will be uploaded tp Pypi"
+python -m twine upload dist/*
```

Comment:
Oh, so on this project the create archive script is used to upload the package to pypi, so we could automate it, nice

Reply:
It is the same as Dakara base. For now, uploading to Pypi requires to enter username and password manually, but it can be automated (but the setup looks non trivial).

Do you think we should split the package creation and package upload in two different scripts?